scripts/teamDataScript.js

```diff
@@ -434,9 +434,8 @@ function sortMatchOutputTable(savedLocStrKeys) //sorts the table by team and the
 
             for (var k = 0; k < innerKeys.length; k++) //displaying answers to each question
             {
```

Comment:
so, if the note that is added is really-long - it has the same scrolling in the input problem as the questions did (good job utilizing the auto-size - it works really well).

But you'll also run into an issue with it forcing the page width really wide. In theory, you could fix this by imposing a max-width, but it would probably be simpler to swap this for a w3-grid as well - except for the fact that you don't know how many items you have at coding time. Think about this and let me know what you come up with.

Reply:
I was tinkering with it and all of the classes added on to the regular table (w3-responsive, w3-table, w3-container etc) didn't have an effect on this problem. I also looked at the w3-grid as well, but you're right, we don't know how many questions the user has and therefore can't assign the divs a class like m6 beforehand.

If it's possible, maybe we can generate classes based on how many questions there are. Like, for the pit table, if there are 6 questions, added on to the column for the team #, the class could be .w3-col.sevenths, with the width being 100/7 % for each column. Or there could be a class called equalWidth and every time the user wants to display the page, the width in the class could be changed based on the # of questions, so the table's width would be adjusted each time the showAllPitTable() method is called (same with the match data table, but probably a different class for that). 

If this is not realistic, I'm thinking that maybe we just impose a max-width element in the table rows, but I'm unsure of how to do that because width:auto; hasn't really been working for me. And if we try to set a specified # of pixels as the max width, that would vary based on screen size.

Reply:
Some more thoughts to throw out there (to make sure we've considered at least most of the reasonable options)

What if we always set everything to s2 (or s(12/len(questions) - whichever is larger) and let it wrap on it's own?
(so, if there were more than 6 questions/items, it would aut-wrap at the sixth, but keep every cell 1/6th - or whatever - of the screen wide). Would it look bad if one of the those had a really long text in it?

What if we turned the table? (using set column widths, but allowing it to scroll to the side)

|Team #|3288|3937|6321|21017|
|---|---|---|---|---|
|Match #|1|12|23|45|
|Balls|2|5|1|3|
|Hatch|1|3|8|2|
|Comments|Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque vel |Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque vel |Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque vel |Lorem ipsum |dolor sit amet, consectetur adipiscing elit. Pellentesque vel |

What if we made it only show the current team and put it where match #'s were "header" rows and everything but comments shared an even space and comments got their own row?

|Balls|Hatch|
|---|---|
|Team: 3288||
|Match: 1||
|1|2|
|Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque vel||
|Match: 9||
|3|4|
|Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque vel||
|Match: 15||
|0|8|
|Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque vel||

(might need to apply some padding to make it line up in a skim-able way)

Reply:
![image](https://user-images.githubusercontent.com/33380236/72100558-72ea9880-32e8-11ea-9437-be1194237693.png)

Reply:
Wait, I solved that problem with `overflow-wrap: break-word`. Here is how it is now:
![image](https://user-images.githubusercontent.com/33380236/72103745-77ff1600-32ef-11ea-9d33-a061cf58b5bc.png)

Looking at this, I think we might need to change the layout, like you were saying.